### PR TITLE
CIT-284 #time 8h start learning javascript

### PR DESCRIPTION
**Pull request by codal-kcolaco**

### CODE REVIEW

File: week_3/15_apr/dom.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+const para = document.querySelector('p')
+console.log(para);
+
+console.log(para.innerText);
+
+para.innerText = "Hi"
+
+const paras = document.querySelectorAll('p')
```
Comment (abhi-codal) [inline week_3/15_apr/dom.js:8]: @codal-kcolaco what is diffrent btw querySelectorAll and querySelector?

File: week_3/15_apr/strings.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+let email = 'kcolaco'
+console.log(email)
+
+let firstName = 'Kevin'
+let lastName = 'Colaco'
+
+let fullName = firstName + ' ' + lastName
```
Comment (abhi-codal) [inline week_3/15_apr/strings.js:7]: avoid using + to concat the strings use.... ${firstName} {lastName} with ``.

File: week_3/15_apr/dom.js (hunk 1 above)
Comment (codal-kcolaco) [reply week_3/15_apr/dom.js:8]: @abhi-codal querySelector selects a single element, while querySelectorAll is used to select multiple elements together. The above code returns a list of all elements with p tags.

**Review by abhi-codal — APPROVED** [review]

File: week_3/15_apr/strings.js (hunk 2 above)
Comment (codal-kcolaco) [reply week_3/15_apr/strings.js:7]: @abhi-codal Yes this was for practice. I have used the ${} in successive files.

**Pull request merged**